scenarios.md

```diff
@@ -101,17 +101,17 @@ Once the developer has locally validated the build, sign, validate scenario, the
 
 1. Locally build and sign an artifact, such as a `net-monitor:abc123` container image
 1. Rename the artifact to reflect the registry it will be pushed to:  
-  `$ docker tag net-monitor:abc123 wabbitnetworks.example.com/networking/net-monitor:1.0`  
-  `$ docker push wabbitnetworks.example.com/networking/net-monitor:1.0`
+  `$ tool tag net-monitor:abc123 wabbitnetworks.example.com/networking/net-monitor:1.0`  
+  `$ tool push wabbitnetworks.example.com/networking/net-monitor:1.0`
 1. Deploy the artifact to a cluster that requires signatures:  
   `$ orchestrator apply -f deploy.yaml`
 1. The orchestrator in the dev environment accepts any signed content, enabling it to trace where deployed artifacts originated from.
```

Comment:
In Scenario #1, I'm using a local docker/moby instance. It could also be **kind**, or other local hosts. Where the validation is out of scope. We could imagine the `docker run` command could support a notary v2 policy. The main aspect of # is no registry is required to create and validate a signature. All the content (image and signature) were created locally.
In this scenario, I'm adding the registry to the scenario. 

But, yes, implementing Notary v2 in all these tools is aspirational. If we get it right, we hope they would. If they don't, we must ask _why_?

In the top picture, you'll see the blue symbols are things that are considered in-scope.